wpilibc/src/main/native/cpp/smartdashboard/ListenerExecutor.cpp

```diff
@@ -0,0 +1,28 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/smartdashboard/ListenerExecutor.h"
+
+using namespace frc;
+
+void ListenerExecutor::Execute(std::function<void()> task) {
```

Comment:
It's not critical for this PR, but there's an argument that executors such as this should use wpi::unique_function instead of std::function so that captures of move-only values in the task function can be accommodated.

Reply:
Noted.  We should keep that in mind for 2021 when we expand the amount of asynchronous support.